src/Database/Model.php

```diff
@@ -529,7 +529,7 @@ public function getObservableEvents()
     /**
      * Get a fresh timestamp for the model.
      *
-     * @return \Winter\Storm\Argon\Argon
+     * @return \Illuminate\Support\Carbon
```

Comment:
Why change this? Isn't Argon more specific than Carbon?

Reply:
It matches Laravel's signature, I believe.